docs/elements/index.md

```diff
@@ -61,4 +61,7 @@ summaries:
   To request a new element or if updates need to be made to an existing element, <a href="https://github.com/RedHat-UX/red-hat-design-system/issues/new/choose" target="_blank">create a GitHub issue</a>.
 {% endsection %}
 
-{% include 'feedback.html' %}
+{% feedback %}
+  <h2>Elements</h2>
```

Comment:
Looks like this one should be "Patterns"

Reply:
It's on the elements page though. @dcaryll do we want the feedback footer for elements to link to patterns and vice versa?